### ISSUE TITLE
Passing ((-) x) as function to map functions is not intuitive

### ISSUE THREAD

**Issue by kuon** [mdgriffith/elm-color-mixing#6]
I noticed that if we do:

```elm
desaturatedColor =
    Color.red
    |> mapSaturation ((-) 0.3)
```

We end up with a color with saturation `-0.7` instead of `0.7`.

I think being able to pass `((-) x)` is good. One workaround would be to chain the passed function with `abs`, it wouldn't break `(*)`, `(+)` and `always` as generating negative makes no real sense (it would change  the behavior of `(+) -0.5` if the value is less than `0.5` but is it an issue?).

**Comment by mdgriffith** [comment]
Interesting, yeah that isn't intuitive.

One potential problem with using `abs` is with something like:

```elm
--existing value of 0.3
 mapSaturation ((-) 0.6)
-- results in 0.3, though it should probably result in 0.
```

Maybe just having a note in the docs to write the full lambda like:

`mapSaturation (\x -> x - 0.6)`

**Comment by kuon** [comment]
Yeah I guess, while `abs` somehow make it intuitive, it has too many drawbacks.

We could suggest the full lambda and also something like:

```elm
sub a b =
    b - a

mapSaturation (sub 0.6)
```

I dunno if any operator makes sense (maybe something like `-:` instead of `sub`, but in the end `sub` is shorter than `(-:)` so...).

But I guess pointing the issue clearly in the doc and provide the example should be enough.

**Comment by kuon** [comment]
Do you want a PR with doc update for this? So we can close that one:)
